projects/smoldot-extension/src/components/IconWeb3.tsx

```diff
@@ -3,13 +3,21 @@ import styled from 'styled-components';
 
 // TODO: more icon-like props
 
-const Icon = styled.span`
+interface Props {
+    size?: string;
+    color?: string;
+}
+
+const Icon = styled('span')<Props>`
```

Comment:
I'm a bit meh about using MUI && styled components. Would stick to just MUI. This is not a blocker

Reply:
Ok. I'll merge it for now and you can refactor that (and this way show me how you prefer it)